docs/source/plugins/using_plugins.rst

```diff
@@ -655,71 +653,63 @@ data, you can access it via the ``result`` property of the returned
         result = await op(...)
         print(result.result) # {...}
 
-.. _delegating-function-calls:
-
-Delegating function calls
--------------------------
+Requesting delegation
+~~~~~~~~~~~~~~~~~~~~~
 
-The
-`@voxel51/utils/delegate <https://github.com/voxel51/fiftyone-plugins/tree/main/plugins/utils>`_
-operator provides a general purpose utility for
-:ref:`delegating execution <delegated-operations>` of an arbitrary function
-call that can be expressed in any of the following forms:
+Operators that support :ref:`delegated execution <delegated-operations>` can
+support this via the `__call__()` syntax by passing the
+`request_delegation=True` flag to
+:func:`execute_operator() <fiftyone.operators.execute_operator>`.
 
--   Execute an arbitrary function: `fcn(*args, **kwargs)`
--   Apply a function to a dataset or view:
-    `fcn(dataset_or_view, *args, **kwargs)`
--   Call an instance method of a dataset or view:
-    `dataset_or_view.fcn(*args, **kwargs)`
-
-Here's some examples of delegating common tasks that can be expressed in the
-above forms:
+In fact, the
```

Comment:
I think we need a kitchen sink example of `__call__``, `request_delegation`, `allow_delegated_execution`, adding a `delegate` property, and determining if an operator was scheduled (which we know is missing). In other words, an example operator that illustrates everything that you should do when implementing `allow_delegated_execution`.